modin/pandas/series_utils.py

```diff
@@ -343,11 +367,22 @@ def repeat(self, repeats):
         return Series(query_compiler=self._query_compiler.str_repeat(repeats))
 
     def rpartition(self, sep=" ", expand=True):
-        return Series(
-            query_compiler=self._query_compiler.str_rpartition(
-                sep=sep, expand=expand
+        if sep is not None and len(sep) == 0:
```

Comment:
can sep be None or should that raise ValueError as well ?

Reply:
+1 to the question. it looks like if `sep` is none, it raises a `TypeError`. 

@helmeleegy can you double check that?

Reply:
In the spirit of trying to keep matching errors, we should try to get this to match.